contracts/payment/PaymentSplitter.sol

```diff
@@ -34,50 +38,52 @@ contract PaymentSplitter {
     }
 
     /**
-     * @dev Payable fallback.
+     * @dev The ether received will be logged with `PaymentReceived` events. Note that these events are not fully
+     * reliable: it's possible for a contract to receive ether without triggering this function. This only affects the
+     * reliability of the events, and not the actual splitting of ether.
      */
     function () external payable {
         emit PaymentReceived(msg.sender, msg.value);
     }
 
     /**
-     * @return the total shares of the contract.
+     * @dev Getter for the total shares held by payees.
      */
     function totalShares() public view returns (uint256) {
         return _totalShares;
     }
 
     /**
-     * @return the total amount already released.
+     * @dev Getter for the total amount of ether already released.
      */
     function totalReleased() public view returns (uint256) {
         return _totalReleased;
     }
 
     /**
-     * @return the shares of an account.
+     * @dev Getter for the amount of shares held by an account.
```

Comment:
Shouldn't this be a payee?

Reply:
The function returns zero if the account is not a payee of the contract. I didn't think it deserved special mention.

Reply:
I thought keeping the wording consistent was a good idea.

Reply:
Oh, you mean replacing "account" with "payee"? Yes I thought the same but the function argument is `account`.

Reply:
Change it :stuck_out_tongue:

Reply:
I actually went through this same process but there's a function called `payee` so we can't.

Reply:
`_payee`?

Reply:
I think it's out of scope for this PR.